src/main/resources/com/google/api/codegen/php/test.snip

```diff
@@ -0,0 +1,119 @@
+@extends "php/common.snip"
+@extends "php/method_sample.snip"
+
+@snippet generate(apiTest)
+    <?php
+    {@licenseSection()}
+
+    namespace {@apiTest.fileHeader.packageName};
+
+    @join import : apiTest.fileHeader.imports
+      @switch import.type
+      @case "SimpleImport"
+        use {@import.fullName};
+      @case "AliasedImport"
+        use {@import.fullName} as {@import.nickname};
+      @default
+        $unhandledCase: {@import.type.toString}$
+      @end
+    @end
+
+    @join serviceImpl : apiTest.serviceImpls
+      class {@serviceImpl.name} extends {@serviceImpl.grpcClassName}
+      {
+        use MockStubTrait;
+      }
+    @end
+
+    @join testClass : apiTest.testClasses
+      class {@testClass.name} extends TestCase
+      {   
+          @join test : testClass.testCases
+            {@testCase(test)}
+    
+          @end
+      }
+    @end
```

Comment:
Since the php code is indented with four spaces, indent the snippet with four spaces too. (Otherwise, it hurtses my eyeses)